Dockerfile

```diff
@@ -8,16 +8,18 @@ MAINTAINER KBase Developer
 
 # -----------------------------------------
 
+ENV BLAST_VERSION='2.12.0+'
+
 COPY ./ /kb/module
 RUN mkdir -p /kb/module/work
 RUN chmod -R a+rw /kb/module
 
 WORKDIR /kb/module
 RUN \
-  curl ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.11.0+-x64-linux.tar.gz > ncbi-blast-2.11.0+-x64-linux.tar.gz && \
-  tar xfz ncbi-blast-2.11.0+-x64-linux.tar.gz && \
-  ln -s ncbi-blast-2.11.0+ blast && \
-  rm -f ncbi-blast-2.11.0+-x64-linux.tar.gz && \
+  curl ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-${BLAST_VERSION}-x64-linux.tar.gz > ncbi-blast-${BLAST_VERSION}-x64-linux.tar.gz && \
```

Comment:
not a big deal, but if you order these statements like so:

```
WORKDIR /kb/module
RUN \
  curl ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-${BLAST_VERSION}-x64-linux.tar.gz > ncbi-blast-${BLAST_VERSION}-x64-linux.tar.gz && \
  # all the rest of the blast install stuff

COPY ./ /kb/module
RUN mkdir -p /kb/module/work && \
  chmod -R a+rw /kb/module && \
  make all
```

it will make your docker builds much quicker in development and testing -- docker caches the image at each build step, and this way you won't be running the blast install every time you make a change to the files in `./`

Reply:
done